dpnp/backend/extensions/lapack/common_helpers.hpp

```diff
@@ -24,9 +24,11 @@
 //*****************************************************************************
 
 #pragma once
+#include <oneapi/mkl.hpp>
```

Comment:
It looks the include never used in that header file. Do we need to add it here?
Probably we need to include `sycl/sycl.hpp` instead? It looks missing.